scripts-available/CDB_EstimateRowCount.sql

```diff
@@ -0,0 +1,31 @@
+-- Internal function to generate stats for a table if they don't exist
+CREATE OR REPLACE FUNCTION _CDB_GenerateStats(reloid REGCLASS)
+RETURNS VOID
+AS $$
+DECLARE
+  has_stats BOOLEAN;
+BEGIN
+  SELECT EXISTS (
+    SELECT * FROM pg_catalog.pg_statistic WHERE starelid = reloid
+  ) INTO has_stats;
+  IF NOT has_stats THEN
+    EXECUTE Format('ANALYZE %s;', reloid);
```

Comment:
Since `reloid` is a REGCLASS here, format %s will properly include the schema and doublequote it if necessary.

Then, I'm relying on `CDB_QueryTablesText` returning schema names as necessary, which seems to be the case. (the results of that function are casted to REGCLASS to be passed here).

So, it seems to work well with org user's, even when the username requires double quotes.